app/sprinkles/admin/templates/forms/user.html.twig

```diff
@@ -24,7 +24,8 @@
                         {% if 'group' in form.fields.disabled %}
                             <input type="text" class="form-control" name="theme" value="{{user.group.name}}" disabled>
                         {% else %}
-                            <select id="input-group" class="form-control js-select2" name="group_id">
+                            <select id="input-group" class="form-control js-select2-group" name="group_id">
+                                <option value="0" </option>
```

Comment:
Is this valid html? Shouldn't it be `<option value="0"></option>` ?

Reply:
Fixed in b26b70512089f7f21cd3d09f609684edf95cf9c5